OSM data program V2.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
```

Comment:
Normally in the beginning of code, please add some readme for the purpose of the notebook. Notebook is supposed to be used for prototype and testing. After the notebook verified your code, it should be commercialised